layouts/partials/job/header.html

```diff
@@ -3,14 +3,15 @@
 
 <div class="h-36 xl:h-80">
   <div class="responsive-container z-10 relative flex items-center h-full">
-    <div class="flex items-center md:items-center flex-row w-full mt-0 md:mt-16 md:mb-12 lg:mb-0 lg:mt-0 ml-2">
-      <div class="w-1/12 xl:w-52 md:h-20 xl:h-52">
+    <div class="flex items-center md:items-center flex-row w-full mt-0 md:mt-16 md:mb-12 lg:mb-0 lg:mt-0">
+      <div class="w-1/12 xl:w-32 md:h-20 xl:h-32">
         <img class="job-icon-header mr-2 xl:mr-8 filter drop-shadow-lg-{{ $role }}" src="/theme-assets/jobs/{{ (lower $role) }}/{{ (lower $job) }}/{{ (lower $job) }}.svg" />
       </div>
-      <h1 class="text-3xl md:text-4xl xl:text-6xl font-semibold filter ml-6 drop-shadow-lg-{{ $role }} mb-0">{{ .Title }}</h1>
+      <h1 class="text-3xl md:text-4xl xl:text-6xl font-semibold filter ml-8 drop-shadow-lg-{{ $role }} mb-0">{{ .Title }}</h1>
     </div>
   </div>
-  <div class="bg-gray-600 absolute top-0 left-0 w-full -z-1 h-96 md:h-72 xl:h-120">
-    <img class="object-cover w-full h-full" src="/theme-assets/jobs/{{ (lower $role) }}/{{ (lower $job) }}/{{ (lower $job) }}_background.png" />
+  <div 
+    class="bg-gray-600 absolute top-0 left-0 w-full -z-1 h-96 md:h-72 xl:h-112 xl:mt-16 bg-bottom md:bg-top bg-cover" 
+    style="background-image: url(/theme-assets/jobs/{{ (lower $role) }}/{{ (lower $job) }}/{{ (lower $job) }}_background.png);">
```

Comment:
going with using background image allows me to use more css options to position, size and adjust the origin of the image compared to using <img>